#! /bin/sh -e
## 61_fix_ru_siteowner.dpatch, by LАszlС 'GCS' BЖszЖrmИnyi <gcs@lsc.hu>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Correct siteowner variable in ru template for newer Mailman versions.

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch ${2:+-d $2}}"

if [ $# -lt 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch $patch_opts -p1 < $0;;
    -unpatch) patch $patch_opts -p1 -R < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1;;
esac

exit 0
@DPATCH@
diff -urNad pkg-mailman/templates/ru/newlist.txt.orig pkg-mailman/templates/ru/newlist.txt
--- pkg-mailman/templates/ru/newlist.txt.orig 2004-09-25 18:05:09.000000000 +0000
+++ pkg-mailman/templates/ru/newlist.txt 2004-09-25 18:05:37.000000000 +0000
@@ -37,4 +37,4 @@
 пользователя.
 
 Все вопросы о системе Вы можете задать, отправив письмо по адресу
-mailman-owner@%(hostname)s.
+    %(siteowner)s.
